.github/workflows/rust.yml

```diff
@@ -175,7 +178,9 @@ jobs:
       - name: Checkout Crate
         uses: actions/checkout@v4
       - name: Checkout Toolchain
-        uses: dtolnay/rust-toolchain@nightly
+        uses: dtolnay/rust-toolchain@master
+        with:
+          toolchain: nightly-2024-02-18
```

Comment:
Can we somehow make it into a variable? Should make it less annoying to keep jobs in sync.

Reply:
Yeah, lemme iterate with the chatbot.

Reply:
Can we call the chatbot HAL 9000? :)

Reply:
Added a file to a repo and read it from there.